draft-fossati-tls-attestation.md

```diff
@@ -841,6 +913,75 @@ possible:
     {{I-D.bft-rats-kat}}.
 
 
+# Cross-protocol binding mechanism
+
+One of the issues that must be addressed when using remote attestation as an
+authentication mechanism is the binding to the outer protocol (i.e., the
+protocol requiring authentication). For every instance of the combined protocol,
+the remote attestation credentials must be verifiably linked to the outer
+protocol. The main reason for this requirement is security: a lack of binding
+can result in the attestation credentials being relayed.
+
+If the attestation credentials can be enhanced freely and in a verifiable way,
+the binding can be performed by inserting the relevant data as new claims. If
+the ways of enhancing the credentials are more restricted, ad-hoc solutions can
+be devised which address the issue. For example, many roots of trust only allow
+a small amount (32-64 bytes) of user-provided data which will be included in the
+attestation token. If more data must be included, it must therefore be
+compressed. In this case, the problem is compounded by the need to also include
+a challenge value coming from the relying party. The verification steps also
+become more complex, as the binding data must be returned from the verifier and
+checked by the relying party.
+
+However, regardless of how the binding and verification are performed, similar
+but distinct approaches need to be taken for every protocol into which remote
```

Comment:
Can you be more specific than the generic word "protocol"? To me the protocol we're discussing here is TLS...

Reply:
A disclaimer would work.